src/rtsp/stream_setup.md

```diff
@@ -0,0 +1,33 @@
+# Receiving a stream using RTSP
+To receive a stream from an RTSP server, at least two separate connections need to be made:
+- one for session negotiation and signalling, using RTSP (usually over TCP)
+- others for stream delivery (one per each stream), using RTP (usually over UDP)
+
+An example setup could look like this:
+![RTSP connection](./rtsp_connection.png)
+
+## Signalling
+The client sends various RTSP requests to the server, the most important ones being DESCRIBE, SETUP and PLAY.
+The server responds to these requests accordingly, depending on the current state of the session.
+
+These responses contain information necessary to set up the delivery of the streams, most importantly RTP mapping,
+the URI to use for control of that particular stream, and FMTP, the media format parameters.
```

Comment:
I think we should be on a little bit higher abstraction level here. In particular, let's not mention RTP mapping or FMTP as we introduce a lot of strange and weird terms. Instead, explain what are  DESCRIBE and PLAY requests. E.g. write that after sending DESCRIBE, server describes itself sending information about codecs it uses, how many media streams it can offfer etc. When you send PLAY server starts sending media.